### PR TITLE
[FEATURE] Explain how to include page TSconfig

### PR DESCRIPTION
**Pull request by linawolf**
On working in the changes regarding TYPO3 12 I noted it was not really explained for 11.5 either

### CODE REVIEW

File: Documentation/ContentMapping/Index.rst [hunk 1]
```diff
@@ -40,19 +40,40 @@ Dynamic Content Rendering in TypoScript
...
+This syntax imports all files ending on `.tsconfig` from the specified folder.
+
+The file :file:`Configuration/page.tsconfig` has to be loaded with an API
+function.
+
+When there is only one site in your installation or all sites share the same
+configurations we suggest to include this file globally:
+
+.. code-block:: php
+   :caption: EXT:site_package/ext_localconf.php
```
Comment (mbrodala) [inline Documentation/ContentMapping/Index.rst:60]:
```suggestion
   :caption: ext_localconf.php
```

We do not use the `EXT:` prefix here anywhere AFAIK.

File: Documentation/ContentMapping/Index.rst [hunk 2]
```diff
@@ -40,19 +40,40 @@ Dynamic Content Rendering in TypoScript
...
+
+   use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+   ExtensionManagementUtility::addPageTSConfig("
+       @import 'EXT:site_package/Configuration/page.tsconfig'
+   ");
```
Comment (mbrodala) [inline Documentation/ContentMapping/Index.rst:66]:
```suggestion
   ExtensionManagementUtility::addPageTSConfig('
       @import "EXT:site_package/Configuration/page.tsconfig"
   ');
```

I think keeping single quotes for PHP code is more important and more likely to be mentioned by CS linters.

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ContentMapping/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ContentMapping/Index.rst:60]: I am using absolute paths that look like this all over the tutorial. This makes it easier to know for unexperienced developers where to make the changes. To us it is very clear, where the file ext_localconf.php should be located to a newcommer it is not

Comment (mbrodala) [reply Documentation/ContentMapping/Index.rst:60]: OK, if it's used consistently in a tutorial, we should stick to this.

**Review by mbrodala — APPROVED** [review]

**Pull request merged**